Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/BaseFocusHandler.cs

```diff
@@ -38,12 +38,18 @@ public virtual bool FocusEnabled
         /// </summary>
         public List<IMixedRealityPointer> Focusers { get; } = new List<IMixedRealityPointer>(0);
 
+        #region IMixedRealityFocusHandler implementation
+
         /// <inheritdoc />
         public virtual void OnFocusEnter(FocusEventData eventData) { }
 
         /// <inheritdoc />
         public virtual void OnFocusExit(FocusEventData eventData) { }
 
+        #endregion IMixedRealityFocusHandler implementation
+
+        #region IMixedRealityFocusChangedHandler Implementation
+
         /// <inheritdoc />
         public virtual void OnBeforeFocusChange(FocusEventData eventData)
```

Comment:
>virtual [](start = 15, length = 7)

This is the reason we got in trouble to begin with. Can we have this and all other 'virtual' in focus handler methods replaced by override, so we get a build error if the interface we're implementing changes?